### PR TITLE
Update colors for accessibility 

### PR DESCRIPTION
**Pull request by matjkel**
Resolves https://github.com/edlio/uniform/issues/137

### CODE REVIEW

File: src/sass/definitions/_colors.scss [hunk 1]
```diff
@@ -29,7 +29,7 @@ $grey-n03      :    rgb(209,210,212);  // #D1D2D4
 $grey-n04      :    rgb(188,189,193);  // #BCBDC1
 $grey-n05      :    rgb(169,170,174);  // #A9AAAE
```
Comment (kenniaa) [inline src/sass/definitions/_colors.scss]: Form hints are marked as using $grey-n05 but they're actually using the label color which uses $grey-n07. :open_mouth: The label color is correct for hints but not for labels. These should be updated so there's no confusion

https://github.com/edlio/uniform/blob/83474de1289dfcdb2e60e4203f09cadb6f745362/src/sass/themes/_default.scss#L95

https://github.com/edlio/uniform/blob/83474de1289dfcdb2e60e4203f09cadb6f745362/src/sass/components/_forms.scss#L15

File: src/sass/definitions/_colors.scss [hunk 2]
```diff
@@ -29,7 +29,7 @@ $grey-n03      :    rgb(209,210,212);  // #D1D2D4
...
 $grey-n06      :    rgb(148,149,153);  // #949599
-$grey-n07      :    rgb(117,118,122);  // #75767A
+$grey-n07      :    rgb(118,118,118);  // #767676
```
Comment (kenniaa) [inline src/sass/definitions/_colors.scss]: Form labels are using $grey-n07 but the ticket requires them to be #444 so the default label color should be updated 
https://github.com/edlio/uniform/blob/83474de1289dfcdb2e60e4203f09cadb6f745362/src/sass/themes/_default.scss#L92

File: src/sass/definitions/_colors.scss [hunk 3]
```diff
@@ -29,7 +29,7 @@ $grey-n03      :    rgb(209,210,212);  // #D1D2D4
 $grey-n04      :    rgb(188,189,193);  // #BCBDC1
```
Comment (kenniaa) [inline src/sass/definitions/_colors.scss]: The ticket wants placeholders to be #767676 but they're using $grey-n04 so the placeholder definition should be updated to the correct color 

https://github.com/edlio/uniform/blob/83474de1289dfcdb2e60e4203f09cadb6f745362/src/sass/themes/_default.scss#L100

File: src/sass/definitions/_colors.scss [hunk 4]
```diff
@@ -29,7 +29,7 @@ $grey-n03      :    rgb(209,210,212);  // #D1D2D4
...
 $grey-n05      :    rgb(169,170,174);  // #A9AAAE
 $grey-n06      :    rgb(148,149,153);  // #949599
```
Comment (kenniaa) [inline src/sass/definitions/_colors.scss]: Input text is using $grey-n06 but the ticket wants them to be #444 so the textfield definition should be updated

https://github.com/edlio/uniform/blob/83474de1289dfcdb2e60e4203f09cadb6f745362/src/sass/themes/_default.scss#L101

File: src/sass/definitions/_colors.scss (hunk 3 above)
Comment (MichaelLiEdlio) [reply src/sass/definitions/_colors.scss]: I don't think we need to worry about `_default.scss`,  https://github.com/edlio/uniform/blob/83474de1289dfcdb2e60e4203f09cadb6f745362/src/sass/themes/cms/_edlio.scss overrides it

File: src/sass/definitions/_colors.scss (hunk 2 above)
Comment (MichaelLiEdlio) [reply src/sass/definitions/_colors.scss]: same as above..

**Comment by kenniaa** [comment]
@MichaelLiEdlio @matjkel let me know if `_edlio.scss` also resolves the remaining comments, so I can resolve them and change the review

**Comment by MichaelLiEdlio** [comment]
all the other ones are fine but I would suggest you put `$label-text-color`  into _edlio.scss and use `$grey-5` instead
 
all we are trying to do is creating a theme color for cms, and later on, if we need to use uniform on other places with an alternate theme default.scss will be the base point

**Comment by matjkel** [comment]
@kenniaa yeah `_edlio.scss` resolves your concerns :) 

@MichaelLiEdlio I took your suggestion and created a hint var as well 👍

**Comment by MichaelLiEdlio** [comment]
all good from me but i believe we need to get the lv2 @cbhernan  approval for this

**Pull request merged**